pkgs/shells/zsh/zplug/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "2.3.4";
+  name = "zplug-${version}";
```

```suggestion
  pname = "zplug";
```

Please consider using ``pname``.

---

Okay, I will do it.